Sources/BraveWallet/Crypto/UnlockWalletView.swift

```diff
@@ -42,101 +50,166 @@ struct UnlockWalletView: View {
+            ) {
+              Text(Strings.Wallet.restoreWalletButtonTitle)
+                .fontWeight(.semibold)
+                .foregroundColor(Color(braveSystemName: .textInteractive))
+                .padding(.vertical, 10)
+                .padding(.horizontal, 20)
+                .frame(maxWidth: .infinity)
             }
           }
-          .padding(.horizontal, 48)
         }
-        VStack(spacing: 30) {
-          Button(action: unlock) {
-            Text(Strings.Wallet.unlockWalletButtonTitle)
-          }
-          .buttonStyle(BraveFilledButtonStyle(size: .normal))
-          .disabled(!isPasswordValid)
-          NavigationLink(destination: RestoreWalletContainerView(keyringStore: keyringStore)) {
-            Text(Strings.Wallet.restoreWalletButtonTitle)
-              .font(.subheadline.weight(.medium))
+        
+        if keyringStore.isKeychainPasswordStored, let icon = biometricsIcon {
+          Button(action: fillPasswordFromKeychain) {
+            icon
+              .imageScale(.large)
+              .font(.headline)
+              .foregroundColor(Color(braveSystemName: .iconInteractive))
+              .padding()
+              .background(Circle()
+                .strokeBorder(Color(braveSystemName: .dividerInteractive), lineWidth: 1))
           }
-          .foregroundColor(Color(.braveLabel))
         }
       }
-      .frame(maxHeight: .infinity, alignment: .top)
-      .padding()
-      .padding(.vertical)
+      .padding(.horizontal, 34)
     }
-    .navigationTitle(Strings.Wallet.cryptoTitle)
-    .navigationBarTitleDisplayMode(.inline)
-    .background(Color(.braveBackground).edgesIgnoringSafeArea(.all))
+    .frame(maxWidth: .infinity, maxHeight: .infinity)
+    .background(
+      Image("wallet-background", bundle: .module)
+        .resizable()
+        .aspectRatio(contentMode: .fill)
+        .edgesIgnoringSafeArea(.all)
+    )
     .onChange(of: password) { _ in
       unlockError = nil
     }
     .onAppear {
+      self.isPasswordFieldFocused = true
+
       DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) { [self] in
         if !keyringStore.lockedManually && !attemptedBiometricsUnlock && keyringStore.defaultKeyring.isLocked && UIApplication.shared.isProtectedDataAvailable {
           attemptedBiometricsUnlock = true
           fillPasswordFromKeychain()
         }
       }
     }
+    .navigationTitle(Strings.Wallet.cryptoTitle)
+    .navigationBarTitleDisplayMode(.inline)
+    .transparentUnlessScrolledNavigationAppearance()
+    .ignoresSafeArea(.keyboard, edges: .bottom)
+  }
+  
+  private var biometricsIcon: Image? {
+    let context = LAContext()
+    if context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil) {
+      switch context.biometryType {
+      case .faceID:
+        return Image(systemName: "faceid")
+      case .touchID:
+        return Image(systemName: "touchid")
+      case .none:
+        return nil
+      @unknown default:
```

Comment:
apparently there is a new enum `.opticID` for iOS 17

Reply:
Nice! Added in [38253b6](https://github.com/brave/brave-ios/pull/8340/commits/38253b69cdba5c2dd83a429a8c68c04b17116b58)